var/spack/repos/builtin/packages/cppad/package.py

```diff
@@ -9,20 +9,32 @@
 class Cppad(CMakePackage):
     """A Package for Differentiation of C++ Algorithms."""
 
+    maintainers("whart222")
+
     homepage = "https://www.coin-or.org/CppAD/"
```

Comment:
That URL contains a link to a page that results in `Not Found`.  Perhaps it would be better to link to the readthdocs page at `https://cppad.readthedocs.io/latest/`?

Reply:
I updated the URL to point to the Github page for cppad.